admin_solutions/index.adoc

```diff
@@ -17,5 +17,6 @@ In this initial version of the guide, you can learn how to:
 - xref:../admin_solutions/user_role_mgmt.adoc#share-templates-cluster[Share templates across the cluster]
 - xref:../admin_solutions/user_role_mgmt.adoc#create-cluster-admin[Create a cluster administrator account]
 - xref:../admin_solutions/authentication.adoc#admin-solutions-authentication[Configure authentication providers]
+- xref:../admin_solutions/certificate_management.adoc[Certificate Management]
```

Comment:
@lpsantil Yes, please. We need to include the anchor ID in the path so that the docs build correctly. Thanks!